201-ospatching-extension-on-ubuntu/README.md

```diff
@@ -0,0 +1,45 @@
+# Simple deployment of an Ubuntu VM with OS Patching extension
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-ospatching-extension-on-ubuntu%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Built by: [Thomas Shao](https://github.com/thomas1206)
```

Comment:
Built by and the table of parameters below should be removed. We will soon render readme's on Azure.com and these will end up being duplicated.